source/application/BioXAS/BioXAS.h

```diff
@@ -8,12 +8,16 @@ namespace BioXAS
 {
 	/// Handles the XRF detector choice.  Available choices are None, KETEK, and 13-element Ge.
 	enum FluorescenceDetector {
-
 		NoneXRFDetector = 0,
 		FourElementVortexDetector = 1,
 		Ge13ElementDetector = 2
 	};
 
+	/// Enum to describe each of the three BioXAS beamlines.
+	enum Beamline {
```

Comment:
I agree. I think Beamline or Endstation is probably more appropriate when considering only Side, Main, and Imaging. I include the FrontEnd that all three share in common here. Not sure what the best name would be.